homeassistant/components/logbook/processor.py

```diff
@@ -398,11 +400,13 @@ def augment(
         data[CONTEXT_DOMAIN] = domain
         event = self.event_cache.get(context_row)
         described = describe_event(event)
-        if name := described.get(ATTR_NAME):
+        if name := described.get(LOGBOOK_ENTRY_NAME):
             data[CONTEXT_NAME] = name
-        if message := described.get(ATTR_MESSAGE):
+        if message := described.get(LOGBOOK_ENTRY_MESSAGE):
             data[CONTEXT_MESSAGE] = message
-        if not (attr_entity_id := described.get(ATTR_ENTITY_ID)):
+        if source := described.get(LOGBOOK_ENTRY_SOURCE):
+            data[CONTEXT_SOURCE] = source
```

Comment:
In the future we can drop sending `CONTEXT_MESSAGE` if `CONTEXT_SOURCE` is available.